vault/tao/community-guidelines.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Life Itself Community Chat Guidelines"
+created: 2023-03-28
+authors: 
+  - nathenfitchen
+---
+
+The Life Itself community chat is a space for members of the Life Itself community to gather online. [Join the community chat here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA) or by scanning the QR Code below. 
+
+![[../assets/images/Whatsapp General Chat QR Code.png |400]]
+
+## Purpose
+
+The Life Itself Community Chat is an online space for members of the Life Itself community for:
+
+1.  👥 Connection - Through the community chat we hope to connect deeply together and develop new relationships. Most importantly we encourage openness for a diversity of perspectives and connecting with people who think and do differently and yet aspire to the same need for change. 
```

Comment:
1️⃣ formatting: 2 spaces are bullet item.

Reply:
This is not clear for me. Do you mean I should add 2 spaces for the numbered list? or something else?